lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+// @ts-nocheck
```

Comment:
how painful is it to have this on (assuming you pass an obj into it instead of a string)?

Reply:
> `// @ts-nocheck`

We shouldn't check in new code that isn't checked (excluding tests...for now:). It becomes a blind spot for navigation, refactors, etc.

I'm sure there are issues since this is deleting stuff off a type defined elsewhere, so I'm happy to help add types if it's super painful.